src/components/CustomerQuota/ItemsSelection/ItemCheckbox.tsx

```diff
@@ -23,8 +25,14 @@ export const ItemCheckbox: FunctionComponent<{
           unit={productQuantity?.unit}
           maxQuantity={maxQuantity}
           accessibilityLabel="item-checkbox"
+          showAddonsToggle={(e) => {
+            e.stopPropagation();
```

Comment:
Nice, so does this cause the checkbox to not be unticked on pressing the toggle?

Reply:
yep! thanks for the idea!